model-engine/model_engine_server/inference/vllm/Dockerfile.vllm

```diff
@@ -9,12 +9,6 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/* \
     apt-get clean
 
-# Need to fix flashinfer at 0.0.8 to support gemma models
```

Comment:
@yunfeng-scale are we sure we don't need this anymore?

Reply:
validated with `google/gemma-2-2b-it`